ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -0,0 +1,289 @@
+SITE_HOST: 'enikshay.in'
+#CAS_SITE_HOST: 'cas.commcarehq.org'
+#ENIKSHAY_SITE_HOST: 'enikshay.commcarehq.org'
+J2ME_SITE_HOST: 'j2me-enikshay.in'
+deploy_env: enikshay
+#tableau_server: 'tableau2.internal.commcarehq.org'
+#TABLEAU_HOST: 'icds.commcarehq.org'
+internal_domain_name: 'enikshay.in'
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: False
+  cchq_http: True
+  cchq_http_redirect: False
+  cchq_http_j2me: True
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech2: False
+  riakcs: True
+#  cas_ssl: True
+#  enikshay_ssl: True
+  couchdb2: False
+
+riak_ring_size: 64
+
+riakcs_s3_access_key: "{{ secrets.RIAKCS_S3_ACCESS_KEY }}"
+riakcs_s3_secret_key: "{{ secrets.RIAKCS_S3_SECRET_KEY }}"
+old_s3_blob_db_access_key: "{{ secrets.OLD_S3_BLOB_DB_ACCESS_KEY }}"
+old_s3_blob_db_secret_key: "{{ secrets.OLD_S3_BLOB_DB_SECRET_KEY }}"
+
+proxy_type: 'nginx'
+
+primary_ssl_env: "enikshay"
+
+ansible_user_password_sha_512: "{{ secrets.ANSIBLE_USER_PASSWORD_SHA_512 }}"
+fake_ssl_cert: no
+
+NEW_RELIC_KEY: "{{ secrets.NEW_RELIC_KEY }}"
```

Comment:
Would be good to get rid of this as NewRelic are phasing out the free server monitoring